router/image/templates/nginx.conf

```diff
@@ -158,6 +158,10 @@ http {
     {{ $useSSL := or (getv "/deis/router/sslCert") "false" }}
     {{ $domains := ls "/deis/domains" }}
     {{ $certs := ls "/deis/certs" }}
+    {{ if exists "/deis/router/affinityArg" }}
+    set_random $prng 0 99;
+    set_if_empty $arg_{{ $affinityArg }} $prng;
```

Comment:
By default it isn't set, which is why I'm using `if exists`. It gets initialized manually with a `deisctl config router set affinityArg=a` for example.

Since it's the name of the `GET` variable containing the affinity token it can be an arbitrary string variable name. So if it were set to empty string or a string that nginx's conf file parser thinks is invalid things would break.